src/components/Math/Math.jsx

```diff
@@ -0,0 +1,53 @@
+import React from "react";
+import PropTypes from "prop-types";
+
+const Math = (props) => {
+  const { first, second, operator, children } = props;
+  let { result } = props;
+  result = Result(props);
+  if (children) {
+    return children(first, second, result);
+  }
+  return (
+    <>
+      <p>
+        {" "}
+        {first} {operator} {second} = {result}{" "}
+      </p>
+    </>
+  );
+};
+
+const Result = (props) => {
+  const { first, second, operator } = props;
+  let { result } = props;
+  switch (operator) {
+    case "+":
+      result = first + second;
+      break;
+    case "-":
+      result = first - second;
+      break;
+    case "/":
+      result = first / second;
+      break;
+    case "*":
+      result = first * second;
+      break;
+    default:
```

Comment:
It is a good practice to keep diff component in diff files in react and the file should have the same name as the component